From eff5e18a39e54ab4853cf60e46b1f55e3cd5745b Mon Sep 17 00:00:00 2001
From: Ryan Lovelett <ryan@lovelett.me>
Date: Thu, 24 Dec 2015 13:10:57 -0500
Subject: [PATCH] Provide a custom preset for Arch Linux

---
 utils/build-presets.ini | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/utils/build-presets.ini b/utils/build-presets.ini
index 43c6890..4e30eec 100644
--- a/utils/build-presets.ini
+++ b/utils/build-presets.ini
@@ -604,6 +604,23 @@ mixin-preset=buildbot_linux
 [preset: buildbot_linux_1404]
 mixin-preset=buildbot_linux
 
+# Arch Linux preset for backwards compat and future customizations.
+[preset: buildbot_arch_linux]
+mixin-preset=buildbot_linux
+
+dash-dash
+
+skip-test-cmark=1
+skip-test-lldb=1
+skip-test-swift=1
+skip-test-llbuild=1
+skip-test-swiftpm=1
+skip-test-xctest=1
+skip-test-foundation=1
+
+reconfigure=1
+verbose-build
+
 [preset: buildbot_linux_armv7]
 release
 llbuild
-- 
2.8.3

