From d2834205785ddf92e1bd736027d025b29f22c76e Mon Sep 17 00:00:00 2001
From: Ryan Lovelett <ryan@lovelett.me>
Date: Thu, 24 Dec 2015 13:10:57 -0500
Subject: [PATCH] Provide a custom preset for Arch Linux

---
 utils/build-presets.ini | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/utils/build-presets.ini b/utils/build-presets.ini
index febfd41..6f5d368 100644
--- a/utils/build-presets.ini
+++ b/utils/build-presets.ini
@@ -587,6 +587,32 @@ mixin-preset=buildbot_linux
 [preset: buildbot_linux_1404]
 mixin-preset=buildbot_linux
 
+# Arch Linux preset for backwards compat and future customizations.
+[preset: buildbot_arch_linux]
+mixin-preset=buildbot_linux
+
+dash-dash
+
+skip-test-cmark=1
+skip-test-lldb=1
+skip-test-swift=1
+skip-test-llbuild=1
+skip-test-swiftpm=1
+skip-test-xctest=1
+skip-test-foundation=1
+skip-test-osx
+skip-test-ios
+skip-test-ios-simulator
+skip-test-tvos
+skip-test-tvos-simulator
+skip-test-watchos
+skip-test-watchos-simulator
+validation-test=0
+skip-test-optimized=1
+
+reconfigure=1
+verbose-build
+
 [preset: buildbot_linux_armv7]
 release
 llbuild
-- 
2.8.3

