From 265e29a5d7fd0b71bee90acc9482cae8851c3ae4 Mon Sep 17 00:00:00 2001
From: Ryan Lovelett <ryan@lovelett.me>
Date: Thu, 24 Dec 2015 13:10:57 -0500
Subject: [PATCH] Provide a custom preset for Arch Linux

---
 utils/build-presets.ini | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/utils/build-presets.ini b/utils/build-presets.ini
index fa74bdd..028e507 100644
--- a/utils/build-presets.ini
+++ b/utils/build-presets.ini
@@ -602,6 +602,32 @@ mixin-preset=buildbot_linux
 [preset: buildbot_linux_1404]
 mixin-preset=buildbot_linux
 
+# Arch Linux preset for backwards compat and future customizations.
+[preset: buildbot_arch_linux]
+mixin-preset=buildbot_linux
+
+dash-dash
+
+skip-test-cmark=1
+skip-test-lldb=1
+skip-test-swift=1
+skip-test-llbuild=1
+skip-test-swiftpm=1
+skip-test-xctest=1
+skip-test-foundation=1
+skip-test-osx=1
+skip-test-ios
+skip-test-ios-simulator
+skip-test-tvos
+skip-test-tvos-simulator
+skip-test-watchos
+skip-test-watchos-simulator
+skip-test-validation=1
+skip-test-optimized=1
+
+reconfigure=1
+verbose-build=1
+
 [preset: buildbot_linux_armv7]
 release
 llbuild
-- 
2.8.0

