From 40ea8c048cebab2d9c056b46d4a09f47d75f38e8 Mon Sep 17 00:00:00 2001
From: Ryan Lovelett <ryan@lovelett.me>
Date: Thu, 24 Dec 2015 13:10:57 -0500
Subject: [PATCH] Provide a custom preset for Arch Linux

---
 utils/build-presets.ini | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/utils/build-presets.ini b/utils/build-presets.ini
index bb7d7b4..ba0b51f 100644
--- a/utils/build-presets.ini
+++ b/utils/build-presets.ini
@@ -482,6 +482,32 @@ mixin-preset=buildbot_linux
 [preset: buildbot_linux_1404]
 mixin-preset=buildbot_linux
 
+# Arch Linux preset for backwards compat and future customizations.
+[preset: buildbot_arch_linux]
+mixin-preset=buildbot_linux
+
+dash-dash
+
+skip-test-cmark=1
+skip-test-lldb=1
+skip-test-swift=1
+skip-test-llbuild=1
+skip-test-swiftpm=1
+skip-test-xctest=1
+skip-test-foundation=1
+skip-test-osx=1
+skip-test-ios=1
+skip-test-ios-simulator=1
+skip-test-tvos=1
+skip-test-tvos-simulator=1
+skip-test-watchos=1
+skip-test-watchos-simulator=1
+skip-test-validation=1
+skip-test-optimized=1
+
+reconfigure=1
+verbose-build=1
+
 [preset: buildbot_linux_armv7]
 release
 llbuild
-- 
2.7.1

